src/scss/_blog.scss

```diff
@@ -0,0 +1,107 @@
+.single-page-title {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 325px;
+
+    .section-container {
+        z-index: 1;
```

Comment:
if you don't define your position: [relative, absolute, fixed], z-index doesn't work